exemption_descriptions.yml

```diff
@@ -1,5 +1,10 @@
 ---
-description:
-    'disclosure-transparency-rules-chapter-5-applies' : "From {exempt_from} the company is exempt from keeping a PSC register because it’s a DTR issuer and the shares are admitted to trading on a regulated market"
+exemption_type:
+    'disclosure-transparency-rules-chapter-five-applies' : "The company has been or is exempt from keeping a PSC register, because it is a DTR issuer and the shares are admitted to trading on a regulated market."
```

Comment:
Can I suggest that changing the underlying value for an enumeration is very problematic for end users. It raises many questions, including:
* Has the value being returned changed, and when will this change happen?
* How should users handle legacy values?
* Does the change imply a change in meaning or behaviour?
There's no problem changing the description of the enumerable, but changing the enumerable itself is fraught with problems, and should be avoided unless it is absolutely essential